lib/api.v2.js

```diff
@@ -112,6 +114,14 @@ function _findOne(collection, query, exactMatch, cb) {
     model = _.first(libs);
   }
 
+  // move etag resource header into place
+  if (exactMatch) {
+    var etagsCollection = dbs[config.etagsCollection]
```

Comment:
Nope - retrieving, that collection is established when the sync operation happens between `api` and `api-sync`